cpp/unreal_plugins/SimulationController/Source/SimulationController/StandaloneComponent.h

```diff
@@ -41,26 +40,5 @@ class StandaloneComponent
     }
 
     TComponent* component_ = nullptr;
-
-private:
-    template <bool is_scene_component>
-    TComponent* createComponent(UObject* owner, AActor* parent, const std::string& name)
-    {
-        SP_ASSERT(false);
-        return nullptr;
-    }
-
-    template <>
-    TComponent* createComponent<true>(UObject* owner, AActor* parent, const std::string& name)
-    {
-        return Unreal::createSceneComponentOutsideOwnerConstructor<TComponent>(actor_, actor_, name);
-    }
-
-    template <>
-    TComponent* createComponent<false>(UObject* owner, AActor* parent, const std::string& name)
-    {
-        return Unreal::createComponentOutsideOwnerConstructor<TComponent>(actor_, name);
-    }
-
```

Comment:
Nice to see this go!

Reply:
Agreed! I always thought this code was painful to look at. Haha.